examples/nlp/fnet_classification_with_keras_hub.py

```diff
@@ -105,41 +113,58 @@
 """
 
 """
-We'll use the `keras.utils.text_dataset_from_directory` utility to generate
-our labelled `tf.data.Dataset` dataset from text files.
+We'll use Pandas to avoid dependency on `keras.utils.text_dataset_from_directory`
```

Comment:
I feel like this adds a lot of code that distracts from the original goal.

You can use torch as a backend and still train with a `tf.data.Dataset`, it just works transparently. You can also train directly with a Pandas series.